torch/optim/lr_scheduler.py

```diff
@@ -374,3 +374,10 @@ def _init_is_better(self, mode, threshold, threshold_mode):
             self.mode_worse = (-float('inf'))
 
         self.is_better = partial(self._cmp, mode, threshold_mode, threshold)
+
+    def state_dict(self):
+        return {key: value for key, value in self.__dict__.items() if key not in {'optimizer', 'is_better'}}
```

Comment:
Can we please avoid reimplementing `state_dict` and do it like this instead?
```python
def state_dict(self):
    state_dict = super(ReduceLROnPlateau, self).state_dict()
    del state_dict['is_better']
    return state_dict
```

Reply:
`ReduceLROnPlateau ` is not inherited from `_LRScheduler` (`object` instead). So I think it's not a reimplementation?

Reply:
Oh I see. Let's leave it as is then!